modules/rostests/apitests/ntdll/NtQueryInformationProcess.c

```diff
@@ -185,6 +185,41 @@ Test_ProcessTimes(void)
 #undef SPIN_TIME
 }
 
+static
+void
+Test_ProcessPriorityClass(void)
```

Comment:
Please rename to something like `Test_ProcessPriorityClassAlignment` to clearly indicate what is being tested here :)